scripts/publish_book.sh

```diff
@@ -0,0 +1,14 @@
+# Exit if any subcommand fails
+set -e
+
+apt-get update
+apt-get -qq install git python-minimal
+python --version
+cargo install mdbook
+cd zokrates_book && mdbook build
+git config --global user.email "stefan.deml+zokratesbot@decentriq.ch"
```

Comment:
Is this required if we use a token?

Reply:
yes, otherwise commits show up with "Unknown" as user.